### PR TITLE
Issue - [DOC-FIX] `model_store` different between torchserve and mlflow-torchserve

### PR DESCRIPTION
**Pull request by shrinath-suresh**
Signed-off-by: Shrinath Suresh <shrinath@ideas2it.com>

## What changes are proposed in this pull request?

[DOC-FIX] `model_store` different between torchserve and mlflow-torchserve

## How is this patch tested?

Existing Unit Tests

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow TorchServe Deployment Plugin users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?
Components
- [ ] `area/deploy`: Main deployment plugin logic
- [ ] `area/build`: Build and test infrastructure for MLflow TorchServe Deployment Plugin
- [x] `area/docs`: MLflow TorchServe Deployment Plugin documentation pages
- [ ] `area/examples`: Example code

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: examples/BertNewsClassification/README.md [hunk 1]
```diff
@@ -82,21 +82,26 @@ create an empty directory `model_store` and run the following command to start t
...
 If needed, version number can also be explicitly mentioned as a config variable.
 
 
 `mlflow deployments create -t torchserve -m file:///home/ubuntu/mlflow-torchserve/examples/BertNewsClassification/models --name news_classification_test -C "VERSION=1.0" -C "MODEL_FILE=news_classifier.py" -C "HANDLER=news_classifier_handler.py"`
 
+Note:
+mlflow-torchserve plugin generates the mar file inside the "model_store" directory. If the `model_store` directory is not present under the current folder, 
+the plugin creates a new directory named "model_store" and generates the mar file inside it.
+
+if the torchserve is already running with a different "model_store" location, ensure to pass the "model_store" path with "--export-path" argument (`--export-path <path-to-model-store>`)
```
Comment (savagej) [inline examples/BertNewsClassification/README.md:104]:
```suggestion
if the torchserve is already running with a different "model_store" location, ensure to pass the "model_store" path with the "EXPORT_PATH" config variable (`-C 'EXPORT_PATH=<torchserve model_store path>'`)
```

File: examples/E2EBert/README.md [hunk 2]
```diff
@@ -96,6 +96,13 @@ If needed, version number can also be explicitly mentioned as a config variable.
 
 `mlflow deployments create -t torchserve -m state_dict.pth --name news_classification_test -C "VERSION=1.0" -C "MODEL_FILE=news_classifier.py" -C "HANDLER=news_classifier_handler.py" -C "EXTRA_FILES=class_mapping.json,bert_base_uncased_vocab.txt"`
 
+Note:
+
+By default, the mlfow-torchserve plugin generates the mar file inside the "model_store" directory. If the model store directory is not present under the current folder, 
+the plugin creates a new directory named "model_store" and generates the mar file inside it.
+
+if the torchserve is already running with a different "model_store" location, ensure to pass the "model_store" path with "--export-path" argument (`--export-path <path-to-model-store>`)
```
Comment (savagej) [inline examples/E2EBert/README.md:104]:
```suggestion
if the torchserve is already running with a different "model_store" location, ensure to pass the "model_store" path with the "EXPORT_PATH" config variable (`-C 'EXPORT_PATH=<torchserve model_store path>'`)
```

File: examples/IrisClassification/README.md [hunk 3]
```diff
@@ -60,6 +60,12 @@ based on the predefined mapping.
 
 `mlflow deployments  create --name iris_test --target torchserve --model-uri iris.pt -C "MODEL_FILE=iris_classification.py" -C "HANDLER=iris_handler.py" -C "EXTRA_FILES=index_to_name.json"`
 
+Note:
+mlflow-torchserve plugin generates the mar file inside the "model_store" directory. If the `model_store` directory is not present under the current folder, 
+the plugin creates a new directory named "model_store" and generates the mar file inside it.
+
+if the torchserve is already running with a different "model_store" location, ensure to pass the "model_store" path with "--export-path" argument (`--export-path <path-to-model-store>`)
```
Comment (savagej) [inline examples/IrisClassification/README.md:67]:
```suggestion
if the torchserve is already running with a different "model_store" location, ensure to pass the "model_store" path with the "EXPORT_PATH" config variable (`-C 'EXPORT_PATH=<torchserve model_store path>'`)
```

File: examples/IrisClassificationTorchScript/README.md [hunk 4]
```diff
@@ -39,6 +39,12 @@ create an empty directory `model_store` and run the following command to start t
 
 `torchserve --start --model-store model_store`
 
+Note:
+mlflow-torchserve plugin generates the mar file inside the "model_store" directory. If the `model_store` directory is not present under the current folder, 
+the plugin creates a new directory named "model_store" and generates the mar file inside it.
+
+if the torchserve is already running with a different "model_store" location, ensure to pass the "model_store" path with "--export-path" argument (`--export-path <path-to-model-store>`)
```
Comment (savagej) [inline examples/IrisClassificationTorchScript/README.md:46]:
```suggestion
if the torchserve is already running with a different "model_store" location, ensure to pass the "model_store" path with the "EXPORT_PATH" config variable (`-C 'EXPORT_PATH=<torchserve model_store path>'`)
```

File: mlflow_torchserve/__init__.py [hunk 5]
```diff
@@ -350,6 +350,11 @@ def __generate_mar_file(
         else:
             model_store = "model_store"
             if not os.path.isdir(model_store):
+                print(
+                    "model_store directory not present in the current folder. "
+                    "Creating model_store directory. To avoid this behaviour"
+                    "pass --export-path <model-store-path> as an input argument"
```
Comment (savagej) [inline mlflow_torchserve/__init__.py:356]:
```suggestion
                     "pass -C 'EXPORT_PATH=<model-store-path> as an input argument"

```

**Review by savagej — COMMENTED** [review]
Only MNIST example has the `--export-path` variable defined in the `create_deployment.py` script.

Unless I've missed a new change you've made recently

**Comment by shrinath-suresh** [comment]
> Only MNIST example has the `--export-path` variable defined in the `create_deployment.py` script.
> 
> Unless I've missed a new change you've made recently

Yes, you are right. I have updated all the readme files

**Comment by shrinath-suresh** [comment]
@savagej Are we good here. Shall we go ahead and merge ?

**Review by savagej — APPROVED** [review]
Great, sorry I didn't reply sooner!

**Review by chauhang — APPROVED** [review]

**Pull request merged**